package.json

```diff
@@ -25,6 +25,7 @@
     "geojsonhint": "0.3.4",
     "gtfs2geojson": "^1.0.0",
     "jsonlint": "1.6.0",
+    "lodash": "^4.17.21",
```

Comment:
Seems like a bit of a piggyback, but certainly a useful one.

Reply:
nice catch, that shouldn't be in there